bookserver/db.py

```diff
@@ -24,6 +24,8 @@
 from .config import settings, BookServerConfig, DatabaseType
 
 
+logger = logging.getLogger(__name__)
```

Comment:
I have a logger for us to use configured within the fastapi framework.  the routers all use this: `from ..applogger import rslogger`. Why not use the same here?

Reply:
Nice! I didn't realize that. I'll update -- thanks.

Reply:
I didn't use `rslogger` in the actual test code, since that seems appropriate, but it's now used in all bookserver code.